bigbang/bigbang_io.py

```diff
@@ -28,7 +28,7 @@
 logger = logging.getLogger(__name__)
 
 
-class ListservMessageIO:
+class MessageIO:
```

Comment:
Is this a wrapper around mailbox.mboxMessage?
Does it generally represent a single email?

It would be good to mention that in these comments.

Maybe even reference the mailbox docs:
https://docs.python.org/3/library/mailbox.html#mboxmessage

Reply:
I added a description on the purpose of MessageIO in its docstring.